modal/_blob_utils.py

```diff
@@ -145,3 +291,18 @@ def get_file_upload_spec(filename: str, mount_filename: str) -> FileUploadSpec:
     return FileUploadSpec(
         filename, mount_filename, use_blob=use_blob, content=content, sha256_hex=sha256_hex, size=size
     )
+
+
+def use_md5(url: str) -> bool:
```

Comment:
Moved this here from modal_utils.blob_utils since it was slightly confusing to have two blob_utils files